train.py

```diff
@@ -54,6 +54,11 @@ def process_args():
                         type=int,
                         default=DEFAULT_NUM_HG,
                         help='number of hourglass blocks')
+    argparser.add_argument('-sub',
```

Comment:
can you add some assertions that the value is in range (0,1)? I can see people thinking it is in a % form rather than decimal form.